Cargo.toml

```
@@ -15,6 +15,9 @@ default = ["alloc"]
 alloc = []
 std = ["alloc"]
 
+[target.'cfg(target_arch = "wasm32")'.dependencies]
```

```suggestion
[target.'cfg(all(target_family = "wasm", target_os = "unknown"))'.dependencies]
```
To make sure this is targeting only `wasm32-unknown-unknown` and not WASI or Emscripten.